### ISSUE TITLE
web3.storage module not found

### ISSUE THREAD

**Issue by hauchu1998** [tkeith/airtracker#1]
Hi @tkeith ,

First I want to thank you for your effort on the project. It's very interesting and amazing. However, when I clone your project and install all the packages. It shows that `Cannot find module 'web3.storage' or its corresponding type declarations.`. Do you mind telling me which node version and webpack you use? I saw someone said the problem might due to this two versions, but I couldn't solve it for now.

Thanks

**Comment by tkeith** [comment]
Hi @hauchu1998 -- I use Node v20.6.0. I don't believe it's dependent on webpack.

Just to confirm -- you installed all the dependencies with "npm install" inside the project directory?

**Comment by hauchu1998** [comment]
yeah, I install all the dependencies with "npm install". Weird thing is that when using Vite rather than Nextjs, the module can be found. Not sure if you met this issue before?
